pkg/cli/cmd/radinit/recipe.go

```diff
@@ -33,9 +30,70 @@ import (
 )
 
 const (
-	DevRecipesRegistry = "ghcr.io/radius-project"
+	// RecipeRepositoryPrefix is the prefix for the repository path.
+	RecipeRepositoryPrefix = "ghcr.io/radius-project/recipes/local-dev/"
```

Comment:
nit: This prefix is tied to local-dev , so you might need to consider naming it as DevRecipeRepositoryPrefix